ur_controllers/src/gpio_controller.cpp

```diff
@@ -499,9 +499,9 @@ bool GPIOController::zeroFTSensor(std_srvs::srv::Trigger::Request::SharedPtr /*r
   // call the service in the hardware
   command_interfaces_[CommandInterfaces::ZERO_FTSENSOR_CMD].set_value(1.0);
 
-  while (command_interfaces_[CommandInterfaces::ZERO_FTSENSOR_ASYNC_SUCCESS].get_value() == ASYNC_WAITING) {
-    // Asynchronous wait until the hardware interface has set the slider value
-    std::this_thread::sleep_for(std::chrono::milliseconds(50));
+  if(!waitForAsyncCommand([&](){return command_interfaces_[CommandInterfaces::IO_ASYNC_SUCCESS].get_value();}))
+  {
+    RCLCPP_WARN(get_node()->get_logger(),"Could not verify that io was set. (This might happen when using the mocked interface)");
```

Comment:
```suggestion
  if(!waitForAsyncCommand([&](){return command_interfaces_[CommandInterfaces::ZERO_FTSENSOR_ASYNC_SUCCESS].get_value();}))
  {
    RCLCPP_WARN(get_node()->get_logger(),"Could not verify that FTS was zeroed. (This might happen when using the mocked interface)");
```